community/csv/src/main/java/org/neo4j/csv/reader/BufferedCharSeeker.java

```diff
@@ -160,6 +145,11 @@ else if ( nextCh != untilChar && !isNewLine( nextCh ) && nextCh != EOF_CHAR )
                 }
                 else if ( isNewLine( ch ) )
                 {   // Found a new line, just keep going
```

Comment:
Oops, forgot to remove that one. Indeed that looks confusing. Love your attention to detail :) keep it up.